src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -71,14 +100,31 @@
                 <p>
                     <ui:outputText value="{!$Label.c.stgHelpEnsureExistContactPreferEmail}" class="slds-text-body--small" />
                 </p>
+                
+                <ui:outputText value="{!$Label.c.stgPreferredPhoneSelection}"/>
+                
             </div>
         </div>
         <div class="slds-col slds-size--1-of-2">
             <lightning:button variant="brand" label="Run Cleanup" iconName="utility:copy" iconPosition="left" onclick="{! c.runCleanUp }" aura:id="cleanUpBtn" disabled="{!or(and(v.hierarchySettings.Disable_Preferred_Email_Enforcement__c , v.isView), !v.isView)}"  />
```

Comment:
Minor and I know it's existing code, but is "Run Cleanup" a label that exists that we can use here instead of hardcoding text?

Reply:
Oh, also - I think the logic that disables this button needs to change. We don't want it to be disabled if preferred email is disabled but preferred phone is enabled, right? (& vice versa)

Reply:
> Oh, also - I think the logic that disables this button needs to change. We don't want it to be disabled if preferred email is disabled but preferred phone is enabled, right? (& vice versa)

Good catch. I will update it such that it is disabled only when both are disabled.